pkg/cmd/release-payload-controller/cmd.go

```diff
@@ -57,12 +59,16 @@ func NewReleasePayloadControllerCommand(name string) *cobra.Command {
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.releaseNamespace, "release-namespace", o.releaseNamespace, "The namespace where ReleasePayloads are stored.")
 	fs.StringVar(&o.jobNamespace, "job-namespace", o.jobNamespace, "The namespace where release creation jobs are configured to run.")
+	fs.StringVar(&o.prowNamespace, "prow-namespace", o.prowNamespace, "The namespace where prow is configured to run.")
```

Comment:
don't we run prow on every build farm?  What if one of them  uses a different namespace?

Reply:
Technically, Prow only runs on `app.ci`.  This particular namespace is where Prow creates the ProwJobs.  It will be on the same machine as the release-payload-controller.